build.zig

```diff
@@ -34,9 +34,9 @@ pub fn build(b: *std.build.Builder) void {
                 .optimize = mode,
             });
             dynamic_lib.linkage = linkage;
-            dynamic_lib.setOutputDir("lib/" ++ platform[1]);
+            // dynamic_lib.setOutputDir("lib/" ++ platform[1]);
```

Comment:
I couldn't yet figure out what the replacement should be here, it might be the cause of the build error.